_posts/2020-06-30-sparklyr-1.3.0-released/sparklyr-1.3.0.Rmd

```diff
@@ -0,0 +1,207 @@
+
+**EMR 6.0:**
+
+This release also features a small but important change that allows sparklyr to correctly connect to the version of Spark 2.4 that is included in Amazon EMR 6.0.
+
+Previously, sparklyr automatically assumes any Spark 2.x it is connecting to is built with Scala 2.11 and attempts to load any required Scala artifacts built with Scala 2.11 as well. This becomes problematic when connecting to Spark 2.4 from Amazon EMR 6.0, which is built with Scala 2.12. Starting from sparklyr 1.3, such problem can be fixed by simply specifying `scala_version = "2.12"` when calling `spark_connect` (e.g., `spark_connect(master = "yarn-client", scala_version = "2.12")`).
+
+**Spark 3.0:**
+
+Last but not least, it is worthwhile to mention sparklyr 1.3.0 is known to be fully compatible with the recently released Spark 3.0. We highly recommend upgrading your copy of sparklyr to 1.3.0 if you plan to have Spark 3.0 as part of your data workflow in future.
+
+## Acknowledgement
+
+In chronological order, we want to thank the following individuals for submitting pull requests towards sparklyr 1.3:
+
+* [Jozef Hajnala](https://github.com/jozefhajnala)
+* [Hossein Falaki](https://github.com/falaki)
+* [Samuel Macêdo](https://github.com/samuelmacedo83)
+* [Yitao Li](https://github.com/yl790)
+* [Andy Zhang](https://github.com/Loquats)
+* [Javier Luraschi](https://github.com/javierluraschi)
+* [Neal Richardson](https://github.com/nealrichardson)
+
+We are also grateful for valuable input on the sparklyr 1.3 roadmap and on [#2434](https://github.com/sparklyr/sparklyr/pull/2434), [#2551](https://github.com/sparklyr/sparklyr/pull/2551), etc from Javier Luraschi, and great spiritual advice on [#1773](https://github.com/sparklyr/sparklyr/issues/1773) and [#2514](https://github.com/sparklyr/sparklyr/issues/2514) from [\@mattpollock](https://github.com/mattpollock) and [\@benmwhite](https://github.com/benmwhite).
+
+Please note if you believe you are missing from the acknowledgement above, it may be because your contribution has been considered part of the next sparklyr release rather than part of the current release. We do make every effort to ensure all contributors are mentioned in this section. In case you believe there is a mistake, please feel free to contact the author of this blog post via e-mail (yitao at rstudio dot com) and request a correction.
```

Comment:
@javierluraschi, @skeydan  Please let me know what you think of this newly added disclaimer. I really couldn't think of better wording for this one. It essentially just says the release branch has to be cut after the last commit in the current release, and before the first commit in the next release :sweat_smile: